docs/src/pages/docs/comparison.md

```diff
@@ -12,40 +12,47 @@ Feature/Capability Key:
+| Bundle Size                                  | [![][bp-react-query]][bpl-react-query] | [![][bp-swr]][bpl-swr]     | [![][bp-apollo]][bpl-apollo]          | [![][bp-react-relay]][bpl-react-relay]|
+| Queries                                      | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Caching                                      | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Devtools                                     | ✅                                     | 🟡                         | ✅                                    | ✅                                    |
+| Polling/Intervals                            | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Parallel Queries                             | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Dependent Queries                            | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Paginated Queries                            | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Infinite Queries                             | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Lagged / "Lazy" Queries<sup>1</sup>          | ✅                                     | 🛑                         | 🛑                                    | ✅                                    |
+| Initial Data                                 | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Scroll Recovery                              | ✅                                     | ✅                         | ✅                                    | 🛑                                    |
+| Cache Manipulation                           | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Outdated Query Dismissal                     | ✅                                     | ✅                         | ✅                                    | ✅                                    |
+| Render Optimization<sup>2</sup>              | ✅                                     | 🛑                         | 🛑                                    | ✅                                    |
```

Comment:
The footnote for this doesn't really explain to me if it will only update a tree of components when a query has changed or if, like Relay, only the individual components that have subscribed to pieces of the result set get re-rendered when their data changes. Which is it?

Reply:
It will only re-render individual components which subscribed to pieces of a changed result. Updates to multiple components are also batched together into one render, not sure if Relay does that?

Reply:
I believe it does, but that would need confirmation. My point is more that the footnote isn’t clear enough to me as to what’s being compared here.

Reply:
Let's lead with a ❔ then?

Reply:
Relay has great render optimisation but I'm not sure about the batched updates. Maybe @kassens from the Relay team could answer this question.